File: 292.Nim-Game.java
// https://leetcode.com/problems/nim-game/
//
// algorithms
// Easy (55.70%)
// Total Accepted:    186,525
// Total Submissions: 334,882
// beats 100.0% of java submissions

public class Solution {
    public boolean canWinNim(int n) {
        if (n % 4 == 0) {
            return false;
        }
        return true;
    }
}

File: 292.Nim-Game.py
# https://leetcode.com/problems/nim-game/
#
# algorithms
# Medium (55.8%)
# Total Accepted:    189,849
# Total Submissions: 340,261


class Solution(object):
    def canWinNim(self, n):
        """
        :type n: int
        :rtype: bool
        """
        if n % 4 == 0:
            return False

        return True
